jujugui/static/gui/src/app/app.js

```diff
@@ -2634,6 +2636,36 @@ YUI.add('juju-gui', function(Y) {
       maasContainer.show();
     },
 
+    // TODO: Move to views/utils.js.
+    showSharing: function(show = true) {
+      const sharing = document.getElementById('sharing-container');
+      // XXX kadams54: temporary until we wire in the actual modelUserInfo API
+      // call.
+      const modelUserInfo = function(callback) {
+        callback(false, [
+          {
+            icon: 'who.png',
```

Comment:
AFAIK we don't have icons, as the ModelUserInfo response is a list of 
```go
type ModelUserInfo struct {
	UserName       string               `json:"user"`
	DisplayName    string               `json:"display-name"`
	LastConnection *time.Time           `json:"last-connection"`
	Access         UserAccessPermission `json:"access"`
}
```
I guess access is "read", "write" or "admin". Also, each result can have an error.
I'd mock the future call like 
```javascript
callback(null, [{
  name: 'who',
  displayName: 'Dr. Who',
  lastConnection: 'now',
  access: 'admin'
}, {
  name: 'dalek',
  displayName: 'Dalek',
  lastConnection: 'never',
  access: 'write',
  err: 'exterminate!'
}]);
```